yt_dlp/extractor/_extractors.py

```diff
@@ -1483,6 +1483,7 @@
 )
 from .pr0gramm import Pr0grammIE
 from .prankcast import PrankCastIE
+from .prankcastpost import PrankCastPostIE
```

Comment:
```suggestion
from .prankcast import PrankCastIE, PrankCastPostIE
```

Move back to same file.